docs/server/source/production-deployment-template-tendermint/workflow.rst

```diff
@@ -0,0 +1,188 @@
+      "bdb-instance-4"
+    ],
+    "NGINX": [
+      "ngx-instance-1",
+      "ngx-instance-2",
+      "ngx-instance-3",
+      "ngx-instance-4"
+    ],
+    "OpenResty": [
+      "openresty-instance-1",
+      "openresty-instance-2",
+      "openresty-instance-3",
+      "openresty-instance-4"
+    ],
+    "MongoDB_Monitoring_Agent": [
+      "mdb-mon-instance-1",
+      "mdb-mon-instance-2",
+      "mdb-mon-instance-3",
+      "mdb-mon-instance-4"
+    ],
+    "Tendermint": [
+      "tendermint-instance-1",
+      "tendermint-instance-2",
+      "tendermint-instance-3",
+      "tendermint-instance-4"
+    ]
+  }
+
+
+☐ Generate three keys and corresponding certificate signing requests (CSRs):
+
+#. Server Certificate for the MongoDB instance
+#. Client Certificate for BigchainDB Server to identify itself to MongoDB
+#. Client Certificate for MongoDB Monitoring Agent to identify itself to MongoDB
+
+Use the self-signed CA to sign those three CSRs:
+
+* Three certificates (one for each CSR).
+
+For help, see the pages:
+
+* :doc:`How to Generate a Server Certificate for MongoDB <../production-deployment-template/server-tls-certificate>`
+* :doc:`How to Generate a Client Certificate for MongoDB <../production-deployment-template/client-tls-certificate>`
+
+☐ Make up an FQDN for your BigchainDB node (e.g. ``mynode.mycorp.com``).
+Make sure you've registered the associated domain name (e.g. ``mycorp.com``),
+and have an SSL certificate for the FQDN.
+(You can get an SSL certificate from any SSL certificate provider.)
+
+☐ Ask the managing organization for the user name to use for authenticating to
+MongoDB.
+
+☐ If the cluster uses 3scale for API authentication, monitoring and billing,
+you must ask the managing organization for all relevant 3scale credentials -
+secret token, service ID, version header and API service token.
+
+☐ If the cluster uses MongoDB Cloud Manager for monitoring,
+you must ask the managing organization for the ``Project ID`` and the
+``Agent API Key``.
+(Each Cloud Manager "Project" has its own ``Project ID``. A ``Project ID`` can
+contain a number of ``Agent API Key`` s. It can be found under
+**Settings**. It was recently added to the Cloud Manager to
+allow easier periodic rotation of the ``Agent API Key`` with a constant
+``Project ID``)
+
+
+.. _generate-the-blockchain-id-and-genesis-time:
+
+3. Generate the Blockchain ID and Genesis Time
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Tendermint nodes require two parameters that need to be common and shared between all the
+participants in the network.
+
+* ``chain_id`` : ID of the blockchain. This must be unique for every blockchain.
+
+  * Example: ``0001-01-01T00:00:00Z``
+
+* ``genesis_time`` : Official time of blockchain start.
+
+  * Example: ``test-chain-9gHylg``
+
+The following parameters can be generated using the ``tendermint init`` command.
+To `initializae <https://tendermint.readthedocs.io/en/master/using-tendermint.html#initialize>`_.
```

Comment:
- Spelling error: it should be "initialize"
- It should read, "To initialize, you will need..."
- "in created" should be "is created"

Reply:
Done.

Reply:
:+1: